{"uuid":"c62b473b-f7f1-4e04-af5f-74eee6bdb7ea","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["c7a501bb-93a7-40ea-b8ff-2bf2845e035d"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715113179,"stop":1696715113179},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715113267,"stop":1696715113268},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715113315,"stop":1696715113316},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715115331,"stop":1696715115331}],"start":1696715113047,"stop":1696715115332}